tools/data_exploration/link_between_var.xml

```diff
@@ -0,0 +1,333 @@
+            </when>
+            <when value="interr">
+                <param name="variable" type="data_column" data_ref="input" numerical="True" label="Select column containing numerical values"/>
+                <param name="variable2" type="data_column" data_ref="input" numerical="True" label="Select column containing numerical values"/>
+                <param name="species" type="data_column" data_ref="input" numerical="False" label="Select column containing species" help="This parameter allows you to divide your scatterplot according to species"/>
+                <param name="variable3" type="data_column" data_ref="input" label="Select column" help="This parameter allows you to divide your scatterplot once more"/>
+            </when>
+            <when value="autocorr">
+                <param name="variable" type="data_column" data_ref="input" numerical="True" label="Select column containing numerical values"/>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output_coli" from_work_dir="Data.txt" format="txt" label="Missing species">
+            <expand macro="explo_filter_colli"/>
+        </data>
+        <data name="output_acp" from_work_dir="valeurs.txt" format="txt" label="Eigen values">
+            <expand macro="explo_filter_pca"/>
+        </data>
+        <data name="output_vif" from_work_dir="vif.tabular" format="tabular" label="Your VIF tabular">
+            <expand macro="explo_filter_vif"/>
+        </data>
+        <data name="output_corr" from_work_dir="corr.tabular" format="tabular" label="Correlation matrix">
+            <expand macro="explo_filter_vif"/>
+        </data>
+        <data name="output_interr" from_work_dir="Species.txt" format="txt" label="Species in data">
+            <expand macro="explo_filter_interr"/>
+        </data>
+        <data name="output_autocorr" from_work_dir="acf.txt" format="txt" label="Acf table">
+            <expand macro="explo_filter_autocorr"/>
+        </data>
+        <collection type="list" name="plots">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" visible="false" format="png"/>
+            <filter>method['type'] != 'vif'</filter>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="Reel_life_survey_fish_modif2.tabular"/>
+            <param name="colnames" value="true"/>
+            <conditional name="method">
+                <param name="type" value="collinearity"/>
+                <param name="species" value="15"/>
+                <param name="columns" value="12,17,18"/>
+            </conditional>
+            <output name="output_coli" value="Missing_species.txt"/>
+            <output_collection name="plots" type="list" count="3">
+  
+            </output_collection>
+        </test>
+    </tests>
+    <expand macro="topic"/>
+    <help><![CDATA[
+=================================
```

Comment:
Maybe give the same names in the Help as it is in the interface when they select the method so that it's easier to find, I'll go through the descriptions to see if the statistical explanations are good for statistics-newbies ;)